inc/conflicts/divi.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Class Optml_Divi
+ *
+ * An example of a conflict.
+ */
+class Optml_Divi extends Optml_abstract_conflict {
+
+	/**
+	 * Optml_Divi constructor.
+	 */
+	public function __construct() {
+		$this->priority = 2;
+		$this->severity = self::SEVERITY_HIGH;
+		parent::__construct();
+	}
+
+	/**
+	 * Set the message property
+	 *
+	 * @since   2.2.6
+	 * @access  public
+	 */
+	public function define_message() {
+		$this->message = sprintf( __( 'It seems your are using %1$sDivi%2$s right now. %3$s In order for Optimole to replace the images in your Divi pages, you will need to go to %4$sDivi -> Theme Options -> Builder -> Advanced -> Static CSS File Generations%5$s and click on Clear for the images to be processed. ', 'optimole-wp' ), '<b>', '</b>', '<br/>', '<a target="_blank" href="' . admin_url( 'admin.php?page=et_divi_options' ) . '">', '</a>' );
+	}
+
+
+	/**
+	 * Determine if conflict is applicable.
+	 *
+	 * @return bool
+	 * @since   2.2.6
+	 * @access  public
+	 */
+	public function is_conflict_valid() {
+
+		if ( ! ( strcmp( wp_get_theme(), 'Divi' ) === 0 ||
```

Comment:
`wp_get_theme()` returns an Object type and you are using it to compare to a string.
Also take note that it might be a child theme of Divi

Reply:
I changed it, it should be ok now